hist/hist/src/TAxis.cxx

```diff
@@ -83,6 +83,14 @@ TAxis::TAxis(Int_t nbins,const Double_t *xbins): TNamed(), TAttAxis()
    Set(nbins,xbins);
 }
 
+TAxis::TAxis(const std::vector<double>& bins):
+  TAxis(bins.size()-1, bins.data())
+{}
+
+TAxis::TAxis(std::vector<double>&& bins):
+  TAxis(bins.size()-1, bins.data())
+{}
```

Comment:
What's the point of the version with the r-value reference if the constructor doesn't take ownership of the vector?